project/platform/src/net/ashald/envfile/platform/ui/EnvFileConfigurationEditor.java

```diff
@@ -108,10 +109,16 @@ public static void writeExternal(@NotNull RunConfigurationBase configuration, @N
     }
 
     public static void patchCommandLine(@NotNull RunConfigurationBase configuration, @Nullable RunnerSettings runnerSettings, @NotNull GeneralCommandLine cmdLine) throws ExecutionException {
-        EnvFileSettings state = configuration.getUserData(USER_DATA_KEY);
-        if (state != null && state.isEnabled()) {
-            Map<String, String> source = cmdLine.getEnvironment();
+        patchEnvironmentVariables(configuration, cmdLine.getEnvironment());
+    }
 
+    public static void patchJavaParameters(@NotNull RunConfigurationBase configuration, JavaParameters params, RunnerSettings runnerSettings) throws ExecutionException {
```

Comment:
The initial design aim for this class was to keep it as generic as possible and use it in concrete implementations. From this perspective rather then keeping methods like `patchCommandLine`  and `patchJavaParameters` I'd rather go with single method `patchEnvironmentVariables` (as you essentially did) and move 1-liners from `patchCommandLine`/`patchJavaParameters` to concrete implementations.